components/Map.js

```diff
@@ -21,6 +21,28 @@ const public_works_streets_url =
 const city_council_districts_url =
   'https://services.arcgis.com/sFnw0xNflSi8J0uh/arcgis/rest/services/City_Council_Districts_View/FeatureServer/0';
 
+const walkable_streets_url =
+  'https://services.arcgis.com/sFnw0xNflSi8J0uh/arcgis/rest/services/BudgetFacilitiesFY2019/FeatureServer/7';
+
+const slow_streets_url =
+  'https://services.arcgis.com/sFnw0xNflSi8J0uh/arcgis/rest/services/BudgetFacilitiesFY2019/FeatureServer/8';
+
+// When a user clicks on a feature, we have to highlight it according to
+// the data type. We use this function below to determine which highlight
+// layer should be used based on what layer a user clicked on.
+const getHighlightLayer = layer => {
+  if (layer == 'publicWorksStreets') {
+    const highlightLayer = 'highlight-line';
+    return highlightLayer;
```

Comment:
These could be simplified to just return the values, rather than assigning them to the `highlightLayer` constant and then returning them.

Reply:
Yes good one thanks!